src/compose.js

```diff
@@ -14,9 +14,9 @@ export default function compose(...funcs) {
     return arg => arg
   }
 
-  if (funcs.length === 1) {
-    return funcs[0]
+  if (funcs.length===1) {
```

Comment:
format: style mismatch:
```suggestion
  if (funcs.length ===1) {
```

`9f25cba4` ␣ should be inserted at column 19.